### PR TITLE
build(webpack): Upgrade loaders in preparation for `webpack@5`

### PR DESCRIPTION
**Pull request by billyvg**
This upgrades some loaders in preparation for upgrading to `webpack@5`

## `mini-css-extract-plugin` [(0.5.0 -> 1.3.9)](https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md)
Better support for webpack 5... only thing that might have affected us (but doesn't)

```
minimum supported Node.js version is 10.13.0
the esModule option is true by default, you need to change const locals = require('./styles.css')/require('./styles.css') on import locals from './styles.css'/import './styles.css''
```

## `style-loader` [(0.23.1 -> 2.0.0)](https://github.com/webpack-contrib/style-loader/blob/master/CHANGELOG.md)
Similar breaking changes as above re: node version and `esModule`.

### CODE REVIEW

File: package.json
Comment (billyvg) [inline package.json:130]: Actually removing these in https://github.com/getsentry/sentry/pull/24582, but will wait until that's merged to rebase this.

**Review by evanpurkhiser — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**
